src/test/java/com/exactpro/junit5/WithByteBufMethodContext.java

```diff
@@ -0,0 +1,93 @@
+package com.exactpro.junit5;
+
+import java.lang.reflect.Method;
+import java.lang.reflect.Parameter;
+import java.util.Collection;
+
+import org.junit.jupiter.api.extension.ParameterContext;
+import org.junit.jupiter.api.extension.ParameterResolutionException;
+import org.junit.jupiter.params.converter.ArgumentConverter;
+import org.junit.jupiter.params.converter.ConvertWith;
+import org.junit.jupiter.params.converter.DefaultArgumentConverter;
+import org.junit.jupiter.params.support.AnnotationConsumerInitializer;
+import org.junit.platform.commons.util.AnnotationUtils;
+import org.junit.platform.commons.util.ReflectionUtils;
+import org.junit.platform.commons.util.StringUtils;
+
+class WithByteBufMethodContext {
+
+    private final WithByteBufMethodContext.Resolver[] resolvers;
+
+    WithByteBufMethodContext(Method testMethod) {
+        Parameter[] parameters = testMethod.getParameters();
+        this.resolvers = new WithByteBufMethodContext.Resolver[parameters.length];
+    }
+
+    boolean hasPotentiallyValidSignature(Parameter[] parameters) {
+        return parameters.length == 1 && parameters[0].getType().equals(Collection.class);
```

Comment:
You left code regarding ConvertWith. In this case all that we can check is that `parameters.length == 1`